azkaban-common/src/main/java/azkaban/executor/ExecutorLoader.java

```diff
@@ -264,7 +268,7 @@ List<ExecutableFlow> fetchFreshFlowsForStatus(final Status status,
       final ImmutableMap<Status, Pair<Duration, String>> validityMap)
       throws ExecutorManagerException;
 
-  List<ExecutableFlow> fetchAgedQueuedFlows(
+  List<Integer> selectAgedQueuedFlows(
```

Comment:
We replace fetchAgedQueuedFlows to selectAgedQueuedFlows and  let other classes to consume